presto-main/src/main/java/io/prestosql/sql/analyzer/StatementAnalyzer.java

```diff
@@ -1280,15 +1280,15 @@ private void analyzeFiltersAndMasks(Table table, QualifiedObjectName name, Optio
 
             ImmutableMap.Builder<Field, List<ViewExpression>> columnMasks = ImmutableMap.builder();
             for (Field field : fields) {
-                if (field.getName().isPresent()) {
-                    List<ViewExpression> masks = accessControl.getColumnMasks(session.toSecurityContext(), name, field.getName().get(), field.getType());
+                field.getName().ifPresent(column -> {
```

Comment:
I find `for` with lambda based `if` with nested `if` as more complex that it used to be. I guess it is because lambda.

Maybe you could extract a method would help here, wdyt?